### DISCUSSION TITLE
zie/ziewin/3.0/ja/doc/qb/infocenter_location

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#75] [Q&A]
# zie/ziewin/3.0/ja/doc/qb/infocenter_location

文書、および他のリソースへのリンクは、 下記アドレスにある、 Z and I Emulator for Windows Information Center で提供されています。 https://help.hcltechsw.com/zie/ziewin/3.0/ja/index.html Information Center には、キーボードのレイアウトやホスト・コード・ページ表など、 ...

https://help.hcl-software.com/zie/ziewin/3.0/ja/doc/qb/infocenter_location.html

**Comment by KunioMatsuo** [comment]
"Information Center" is IBM term.  This should be like "HCLSoftware Product Documentation".
